src/components/SubCategoryDetails/SubCategoryDetails.scss

```diff
@@ -0,0 +1,45 @@
+@import '~style/variables';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding-top: 32px;
+  width: 100%;
+  margin: 0 0 16px;
+  background: $color-white;
+  padding-left: 16px;
+  overflow: hidden;
+
+  @include for-tablet-landscape-up {
+    padding-left: 32px;
+  }
+}
+
+.heading {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.chartGrid {
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.chartWrapper {
+  flex-basis: 49%;
+
+  @include for-phone-only {
+    flex-basis: 100%;
+  }
+}
+
+.heroChartWrapper {
+  flex-basis: 100%;
+}
```

Comment:
Unless I'm missing something, there's no need for `flex-basis` here, all you want to do is set the width, right? (`flex-basis` defaults to `auto` which just resolves to the `width`. Interestingly, the spec actually recommends against using flex-basis directly.)

Also you can do 50% not 49%.

Also, if you're setting, then overriding a value, prefer 'mobile first'.

You can do:
```scss
.chartWrapper {
  width: 100%;

  @include for-tablet-portrait-up {
    width: 50%;
  }
}

.heroChartWrapper {
  width: 100%;
}
```

Also, the axis labels are cut off (very neatly). See 0%, 5%, 0%, 5% in the screenshot.

Reply:
The layout here looks a bit funky when one chart has a legend. Also the title is indented in quite a way and looks strange. Also I wanted to do something with big red arrows.
![image](https://user-images.githubusercontent.com/4443482/31645201-7e4ef1fe-b346-11e7-8181-3e458bc85d3f.png)